unittest/Dialect/TritonGPU/LinearLayoutConversionsTest.cpp

```diff
@@ -881,6 +881,167 @@ TEST_F(LinearLayoutConversionsTest, warp1onK_mfma16_rhs_kwidth8) {
                          {S("dim0"), S("dim1"), S("dim2")}));
 }
 
+TEST_F(LinearLayoutConversionsTest, mfma32_dot_op_lhs) {
```

Comment:
We are missing rhs kWidth=8 tests?

Reply:
We have some configurations tested, I can extend it